jujugui/static/gui/src/app/components/generic-button/generic-button.js

```diff
@@ -75,7 +75,9 @@ GenericButton.propTypes = {
   type: PropTypes.string
 };
 
-GenericButton.displayName = 'GenericButton';
+if (module) {
+  module.exports = GenericButton;
```

Comment:
I assume this is a temporary measure until all the components have been updated to require GenericButton?

Reply:
Ah, I see your comment below.